daprdocs/content/en/developing-applications/building-blocks/service-invocation/howto-invoke-services-grpc.md

```diff
@@ -0,0 +1,238 @@
+---
+type: docs
+title: "How-To: Invoke services using gRPC"
+linkTitle: "How-To: Invoke services using gRPC"
```

Comment:
Could you update the file "title: "How-To: Invoke services using HTTP" file and change its title from linkTitle: "How-To: Invoke services" to "How-To: Invoke services using HTTP" as part of this change for consistency

Reply:
Done

Reply:
No, the link title needs to stay short. See my comment about the HTTP one. This can be "How To: gRPC Invocation"

Reply:
Changed, see below.